services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/OpenHouseInternalRepositoryImpl.java

```diff
@@ -513,6 +513,14 @@ public void deleteById(TableDtoPrimaryKey tableDtoPrimaryKey) {
         true);
```

Comment:
Isn't purge defaulted to true here, Does this need to change?

Reply:
On Line 518 I have a function for soft delete `softDeleteById`, which calls catalog.dropTable with purge false.